lib/xcode.js

```diff
@@ -229,6 +229,31 @@ const getMaxIOSSDK = _.memoize(
   }
 );
 
+async function getMaxTVOSSDKWithoutRetry (timeout = XCRUN_TIMEOUT) {
+  const version = await getVersion(false, DEFAULT_NUMBER_OF_RETRIES, timeout);
+  if (version[0] === '4') {
+    return '6.1';
+  }
+
+  const args = ['--sdk',  'appletvsimulator',  '--show-sdk-version'];
+  const {stdout} = await runXcrunCommand(args, timeout);
+
+  const sdkVersion = stdout.trim();
+  const match = /\d.\d/.exec(stdout);
+
+  if (!match) {
```

Comment:
To me it would be more intuitive to just use `parseFloat`:
```js
if (isNaN(parseFloat(sdkVersion))) {
  throw new Error(`xcrun returned a non-numeric tvOS SDK version: '${sdkVersion}'`);
}
```